quicktests/overlaying/tests/realistic/baseball.js

```diff
@@ -6,7 +6,7 @@ function makeData() {
 function run(svg, data, Plottable) {
   "use strict";
 
-  d3.csv("../overlaying/tests/baseball.csv").get(function(error, rows) {
+  d3.csv("/quicktests/overlaying/data/baseball.csv").get(function(error, rows) {
```

Comment:
Oh the root directory is considered to be the directory that `grunt` is run from?

Wait... that seems incorrect to me (What happens if I run `grunt` on a directory that contains Plottable)